src/components/Request/Pay/Views/Success.view.tsx

```diff
@@ -18,10 +21,32 @@ export const SuccessView = ({ transactionHash, requestLinkData }: _consts.IPaySc
     const explorerUrlAxelarWithTx = 'https://axelarscan.io/gmp/' + transactionHash
 
     useEffect(() => {
-        if (transactionHash) {
+        if (explorerUrlDestChainWithTxHash) {
+            peanut.submitRequestLinkFulfillment({
```

Comment:
thought: i wonder if we can combine this with the x-chain claim functionality - seems like a good place for DRY